packages/block-editor/src/components/block-list/block.js

```diff
@@ -466,8 +466,15 @@ function BlockListBlock( {
 			aria-label={ blockLabel }
 			childHandledEvents={ [ 'onDragStart', 'onMouseDown' ] }
 			tagName={ animated.div }
-			style={ style }
 			{ ...blockWrapperProps }
+			style={
+				wrapperProps && wrapperProps.style ?
```

Comment:
@youknowriad Question: Is it necessary to add `animationStyle` even if `enableAnimation` is set to `false`? This way it's overriding the potential `transform` style added by `wrapperProps.style`.

Reply:
Probably not necessary in that case.

Reply:
OK, thanks, will create a PR.